nestkernel/kernel_manager.cpp

```diff
@@ -73,6 +77,7 @@ nest::KernelManager::KernelManager()
       &io_manager,
       &node_manager } )
```

Comment:
It should be possible to get guarantees about the order of initialisation here—could we put model_manager last to avoid having to use the ``builtin_models_registered_`` flag?

Reply:
The sequence in the file is the result of quite a bit of trial and error. Every change I've ever tried lead to segmentation faults somewhere, so I think the current solution is the only one feasible right now.

The proper solution would of course be to assert the sequence by a clear programmatic definition of the dependencies between the different managers and maybe the architecture in general. However, I'm afraid that that's for another life (aka NEST 4 :wink:)